### PR TITLE
CCT-738 Order API docs updates

### PR DESCRIPTION
**Pull request by arius86**

### CODE REVIEW

File: source/reference/stable/order-api/get-orders.md [hunk 1]
```diff
@@ -215,44 +215,87 @@ This will fetch the orders allowed for the plugin being set up.
          "grandTotalValue": 3978.75,
          "grandTotalTaxValue": 795.75,
          "internalComment": "",
-         "otherComment": "",
```
Comment (fransr) [inline source/reference/stable/order-api/get-orders.md]: This one should still be here, right?

Comment (arius86) [reply source/reference/stable/order-api/get-orders.md]: Yes! I removed it by accident, good catch 👍

File: source/reference/stable/order-api/get-orders.md [hunk 2]
```diff
@@ -215,44 +215,89 @@ This will fetch the orders allowed for the plugin being set up.
...
          "products": [
            {
              "lineId": "43243",
              "sku": "S123K345U1",
              "variantSku": "",
-             "sizeSku": "",
+             "sizeSku": ""
```
Comment (jacor84) [inline source/reference/stable/order-api/get-orders.md]: Without this coma it is no longer valid. Why? 🤔

**Comment by arius86** [comment]
Applied all changes @fransr @jacor84

**Pull request merged**